### DISCUSSION TITLE
Icon changes after Lsp load?

### DISCUSSION THREAD

**Discussion by vunhatchuong** [Bekaboo/dropbar.nvim#206] [Q&A] [answered]
Before Lsp load these are the icons:

![image](https://github.com/user-attachments/assets/9b0eb1a2-8ae4-46e8-920a-acd7e0370a57)

But after that it returns these ugly one:

![image](https://github.com/user-attachments/assets/86d4341a-fb5d-48b8-b994-4a0b8bf1da36)

How do I set to use the one before Lsp load permanently?

**Comment by Bekaboo** [comment, marked as answer]
You can change `opts.icons.kinds.symbols` for the symbol icons, see https://github.com/Bekaboo/dropbar.nvim/blob/master/README.md#icons

**Reply by vunhatchuong** [reply]
Thanks for the answer, I should mention I know that I can change the symbols in there but it's just strange to me that there's no default and it actually fallback to a less desirable one in my mind.

**Reply by Bekaboo** [reply]
The reason why the icons change is that `treesitter` and `lsp` are not guaranteed to return symbols with the exact same name and contents, thus the icon matched for them can be different.

> it actually fallback to a less desirable one in my mind.

That's why the option `opts.icons.kinds.symbols` exists. People have different preferences, if the default option value is not desirable you can just set the option to whatever you like.
